chart/app-readme.md

```diff
@@ -18,14 +18,16 @@ This chart installs the following components:
-| CIS    | eks                     | eks-1.2.0                                                                                                          | eks                 |
-| CIS    | aks                     | aks-1.0                                                                                                            | aks                 |
-| CIS    | gke                     | gke-1.2.0                                                                                                          | gke                 |
-| CIS    | gke                     | gke-1.6.0                                                                                                          | gke-1.29+            |
+| CIS    | any                     | cis-1.9                                                                    | v1.27+              |
+| CIS    | any                     | cis-1.8                                                                    | v1.26               |
+| CIS    | rke                     | [rke-cis-1.8-permissive](https://github.com/rancher/security-scan/tree/release/v0.4/package/cfg/rke-cis-1.8-permissive)  | rke1-v1.26+         |
+| CIS    | rke                     | [rke-cis-1.8-hardened](https://github.com/rancher/security-scan/tree/release/v0.4/package/cfg/rke-cis-1.8-hardened)                  | rke1-v1.26+         |
+| CIS    | rke2                    | [rke2-cis-1.8-permissive](https://github.com/rancher/security-scan/tree/release/v0.4/package/cfg/rke2-cis-1.8-permissive) | rke2-v1.26+         |
+| CIS    | rke2                    | [rke2-cis-1.8-hardened](https://github.com/rancher/security-scan/tree/release/v0.4/package/cfg/rke2-cis-1.8-hardened)                 | rke2-v1.26+         |
+| CIS    | k3s                     | [k3s-cis-1.9](https://github.com/rancher/security-scan/tree/release/v0.4/package/cfg/k3s-cis-1.9)                                | k3s-v1.27+          |
+| CIS    | k3s                     | [k3s-cis-1.8-permissive](https://github.com/rancher/security-scan/tree/release/v0.4/package/cfg/k3s-cis-1.8-permissive)  | k3s-v1.26           |
+| CIS    | k3s                     | [k3s-cis-1.8-hardened](https://github.com/rancher/security-scan/tree/release/v0.4/package/cfg/k3s-cis-1.8-hardened)                  | k3s-v1.26           |
+| CIS    | eks                     | eks-1.2.0                                                                    | eks                 |
+| CIS    | aks                     | aks-1.0                                                                    | aks                 |
+| CIS    | gke                     | gke-1.2.0                                                                    | gke                 |
+| CIS    | gke                     | gke-1.6.0                                                                    | gke-1.29+            |
```

Comment:
@vardhaman22 could you add the following kube-bench links to the cfgs ? Added also the corresponding targeted version for `gke-1.2.0`. Note that for `eks` and `aks`, no versions are specified in kube-bench nor in CIS Workbench.

```suggestion
| CIS    | any                     | [cis-1.9](https://github.com/aquasecurity/kube-bench/tree/main/cfg/cis-1.9)                                                                    | v1.27+             |
| CIS    | any                     | [cis-1.8](https://github.com/aquasecurity/kube-bench/tree/main/cfg/cis-1.8)                                                                    | v1.26               |
| CIS    | rke                     | [rke-cis-1.8-permissive](https://github.com/rancher/security-scan/tree/release/v0.4/package/cfg/rke-cis-1.8-permissive)  | rke1-v1.26+         |
| CIS    | rke                     | [rke-cis-1.8-hardened](https://github.com/rancher/security-scan/tree/release/v0.4/package/cfg/rke-cis-1.8-hardened)                  | rke1-v1.26+         |
| CIS    | rke2                    | [rke2-cis-1.8-permissive](https://github.com/rancher/security-scan/tree/release/v0.4/package/cfg/rke2-cis-1.8-permissive) | rke2-v1.26+         |
| CIS    | rke2                    | [rke2-cis-1.8-hardened](https://github.com/rancher/security-scan/tree/release/v0.4/package/cfg/rke2-cis-1.8-hardened)                 | rke2-v1.26+         |
| CIS    | k3s                     | [k3s-cis-1.9](https://github.com/rancher/security-scan/tree/release/v0.4/package/cfg/k3s-cis-1.9)                                | k3s-v1.27+          |
| CIS    | k3s                     | [k3s-cis-1.8-permissive](https://github.com/rancher/security-scan/tree/release/v0.4/package/cfg/k3s-cis-1.8-permissive)  | k3s-v1.26           |
| CIS    | k3s                     | [k3s-cis-1.8-hardened](https://github.com/rancher/security-scan/tree/release/v0.4/package/cfg/k3s-cis-1.8-hardened)                  | k3s-v1.26           |
| CIS    | eks                     | [eks-1.2.0 ](https://github.com/aquasecurity/kube-bench/tree/main/cfg/eks-1.2.0)                                                                   | eks                 |
| CIS    | aks                     | [aks-1.0](https://github.com/aquasecurity/kube-bench/tree/main/cfg/aks-1.0)                                                                    | aks                 |
| CIS    | gke                     | [gke-1.2.0](https://github.com/aquasecurity/kube-bench/tree/main/cfg/gke-1.2.0)                                                                    | gke-1.20                 |
| CIS    | gke                     | [gke-1.6.0](https://github.com/aquasecurity/kube-bench/tree/main/cfg/gke-1.6.0)                                                                    | gke-1.29+            |
```

Reply:
done